src/main/java/com/hubspot/jinjava/lib/filter/SliceFilter.java

```diff
@@ -47,7 +51,30 @@ public Object filter(Object var, JinjavaInterpreter interpreter, String... args)
     }
 
     int slices = NumberUtils.toInt(args[0], 3);
```

Comment:
Any reason we default to `3` here?

Reply:
Ideally we would throw an error if `args[0]` wasn't provided, however we can't do that without breaking existing functionality. Somebody could be using a raw `|slice` out there.